Translators/Incident/main.py

```diff
@@ -0,0 +1,111 @@
+import json
+import requests
+import logging
+import os
+from pgquery import query_db
+from tim_generator import get_action, get_effect, get_point, get_itis_codes, calculate_direction
+from flask import request, Flask
+from datetime import datetime
+
+app = Flask(__name__)
+
+log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
+logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
+
+def translate(incident_geojson):
+    tims = {"timIncidentList": []}
+
+    for feature in incident_geojson["features"]:
+        tim_body = {}
+        tim_body["clientId"] = feature["properties"]["id"].replace("_", "-")
+        tim_body["incidentId"] = feature["properties"]["id"].replace("_", "-")
+        if feature["geometry"]["type"] == "Point":
+            tim_body["startPoint"] = get_point(feature["geometry"]["coordinates"])
+            tim_body["endPoint"] = get_point(feature["geometry"]["coordinates"])
+            tim_body["direction"] = "I"
+        else:
+            tim_body["startPoint"] = get_point(feature["geometry"]["coordinates"][0])
+            tim_body["endPoint"] = get_point(feature["geometry"]["coordinates"][-1])
+            tim_body["direction"] = calculate_direction(feature['geometry']['coordinates'])
+        tim_body["route"] = feature["properties"]["routeName"].replace("_", "-")
+        tim_body["highway"] = feature["properties"]["routeName"].replace("_", "-")
+        tim_body["problem"] = feature["properties"]["type"]

```

Comment:
(non-blocking) Using a model class for this 'feature' object would make the code much easier to digest.

For example, this line could be rewritten as the following:
```
tim_body["problem"] = feature.get_problem_type()
```

Reply:
I've added an IncidentFeature class for this